### DISCUSSION TITLE
Push Denied (403) to Public Repo of Which I am a Collaborator, HTTPS only

### DISCUSSION THREAD

**Discussion by PikachuEXE** [community/community#22298] [New to GitHub]
<p>I use HTTPS URLs as remotes by default</p>
<p>I can push to my own repos</p>
<p>I can push to private repos</p>
<p>I can push to most public repos where I am Collaborator (except this one)</p>
<p>The repo that I cannot push via HTTPS is:</p>
<p><a href="https://github.com/AssetSync/asset_sync" rel="nofollow noopener">https://github.com/AssetSync/asset_sync</a></p>
<p>But I can push via SSH</p>
<p>I was able to push via HTTPS to that repo about 2 months ago</p>
<p>There is no protected branch (checked project settings)</p>
<p>There should be no permission issue (I can still push via SSH)</p>
<p>I have read some similar posts already</p>
<p><a href="https://github.community/t/push-denied-to-private-repo-of-which-i-am-a-collaborator/1721">https://github.community/t5/How-to-use-Git-and-GitHub/Push-Denied-to-Private-Repo-of-Which-I-am-a-Collaborator/m-p/18649</a></p>
<p><a href="https://github.community/t/git-push/500">https://github.community/t5/How-to-use-Git-and-GitHub/git-push/m-p/2479</a></p>
<p>Thanks for any help provided in advance</p>

**Comment by lee-dohm** [comment]
<p>For privacy and security reasons, the forum staff doesn’t have access to how your account or the repository is configured. If you’re still having problems with this, you should reach out to private support so they can take a look at things behind the scenes.</p>
<p>Let us know if you have more questions.</p>

**Comment by PikachuEXE** [comment]
<p>Thanks for your advice.</p>
<p>I will reach private support and see what can be done</p>

**Comment by PikachuEXE** [comment, marked as answer]
<p>Private Support responded saying there is </p>
<p>Third-party application access policy in the organization</p>
<p>And my git client is SourceTree which is using OAuth</p>
<p><a href="https://help.github.com/en/articles/about-oauth-app-access-restrictions" rel="nofollow noopener">https://help.github.com/en/articles/about-oauth-app-access-restrictions</a></p>
<p>So since I am owner I just grant the organization access to SourceTree to solve this issue</p>
